ee/tabby-webserver/src/schema/github_repository_provider.rs

```diff
@@ -11,6 +11,19 @@ pub struct GithubRepositoryProvider {
     pub id: ID,
     pub display_name: String,
     pub application_id: String,
+    /// Will never be returned to the client / from GraphQL endpoints.
+    /// Make sure to strip using [Self::strip_access_token].
+    pub access_token: Option<String>,
```

Comment:
```suggestion
   #[graphql(skip)]
    pub access_token: Option<String>,
```

Just realized we have better way of doing this! You might remove `strip_access_token` as well

Reply:
Very good to know! I was a bit worried about that